core-client/transports/src/transports/http.rs

```diff
@@ -49,6 +49,10 @@ where
 					log::warn!("Unsupported `RpcMessage` type `Subscribe`.");
 					return None;
 				}
+				RpcMessage::Notify(_) => {
```

Comment:
Why are notifications not supported by HTTP client?

Reply:
I had assumed that notifications weren't feasible to support for the same reasons that subscriptions are unsupported, so I left it out of this pull request for the time being for the sake of having an MVP. With that said, I'm willing to take a stab at it in this PR if you think nothing is blocking it.